### ISSUE TITLE
How to detect change event?

### ISSUE THREAD

**Issue by decadenza** [DashboardCode/BsMultiSelect#1]
Hi, I tried to detect the change event on the original select using jQuery.change() but it's not working.
Any solution to detect a change of the selected options?

**Comment by rpokrovskij** [comment]
Hi, 
you are right: there are no trigger event call. I will add it.

**Comment by rpokrovskij** [comment]
or if it will nor work since original select is hidden (display: none)  I will add an event to plugin.

**Comment by rpokrovskij** [comment]
done. published. trigger change on disabled works (tested on Edge)

**Comment by Noido** [comment]
Hi  can you show simple example of change event plz ? No doc for that 

Thanks

**Comment by rpokrovskij** [comment]
you should use original `select` change event: `$( "select" ) .change(function ()` and so on

**Comment by oliverfurmage** [comment]
Change event doesn't seem to be triggered for me when you select on option. However when you remove an object the change event does trigger.
```
    $("select").change(function(e){
      triggerFilter($(this).attr("data-filter-by"), e.target.options);
    });
```

**Comment by Noido** [comment]
Same for me @oliverfurmage when i remove and object change event does trigger but not other, i don t understand . @rpokrovskij say it works but apparently not

**Comment by rpokrovskij** [comment]
confirmed, "change" works only on unselect.

**Comment by rpokrovskij** [comment]
resolved in version 0.2.21
